File: leetcode/excel-sheet-column-number.go
package main

func titleToNumber(columnTitle string) int {
	var number int
	for i, multiple := len(columnTitle)-1, 1; i >= 0; i-- {
		k := columnTitle[i] - 'A' + 1
		number = number + int(k)*multiple
		multiple = multiple * 26
	}
	return number
}

File: leetcode/excel-sheet-column-number.py
class Solution:
    def titleToNumber(self, columnTitle: str) -> int:
        number, multiple = 0, 1
        for i in range(len(columnTitle) - 1, -1, -1):
            number += (ord(columnTitle[i]) - ord("A") + 1) * multiple
            multiple *= 26
        return number
